projects/storefrontapp-e2e-cypress/cypress/e2e/regression/coupons/my-coupons.e2e-spec-flaky.cy.ts

```diff
@@ -34,6 +34,19 @@ viewportContext(['mobile', 'desktop'], () => {
     });
   });
 
+  describe('My coupons - claim coupons with code in body using authenticated user', () => {
+    beforeEach(() => {
+      cy.window().then((win) => {
+        win.sessionStorage.clear();
+      });
+      cy.requireLoggedIn();
+    });
+    //TODO uncomment once ClaimCustomerCouponWithCodeInBody works in the backend,the new Occ endpoint is available since Commerce 2211.28.
```

Comment:
hi @rmch91,  disable toggle and skip new added cases for now, not sure when and who should  enable the cases?
Thanks.